src/polyfill.js

```diff
@@ -0,0 +1,7 @@
+let global = Function('return this')();
```

Comment:
i think here you can just use `global`; a bundler will convert it appropriately for you - unless you're worried about someone reassigning the global global.